docs/finality-provider-operation.md

```diff
@@ -646,8 +648,12 @@ Required parameters:
   instance that the finality provider should use. If one is not provided the
   finality provider will request the creation of a new one from the connected
   EOTS manager instance.
-* `--commission`: The commission rate (between 0 and 1) that you'll receive from
+* `--commission-rate`: The commission rate (between 0 and 1) that you'll receive from
   delegators
+* `--commission-max-rate`: The maximum commission rate (between 0 and 1) that
+  you'll receive from delegators
+* `--commission-max-change-rate`: The maximum commission change rate (between 0 and 1)
+  that you'll receive from delegators
```

Comment:
Let's follow the description here https://github.com/cosmos/cosmos-sdk/blob/46be66ed4205bf109d93704dd14c0822d048972e/x/staking/client/cli/flags.go#L56